### PR TITLE
{tools}[GCCcore/13.3.0] poetry v1.8.3, hatchling v1.24.2, virtualenv v20.26.2, ...

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/02c79eb506e1b6e60460ba8ee8b5cbff for a full test report.

File: easybuild/easyconfigs/p/poetry/poetry-1.8.3-GCCcore-13.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    ('poetry-core', '1.9.0', {
+        'modulename': 'poetry.core',
+        'sources': ['poetry_core-%(version)s.tar.gz'],
+        'checksums': ['fa7a4001eae8aa572ee84f35feb510b321bd652e5cf9293249d62853e1f935a2'],
+    }),
+    ('poetry-plugin-export', '1.8.0', {
+        'sources': ['poetry_plugin_export-%(version)s.tar.gz'],
+        'checksums': ['1fa6168a85d59395d835ca564bc19862a7c76061e60c3e7dfaec70d50937fc61'],
+    }),
+    ('fastjsonschema', '2.19.1'),
```
Comment (Micket) [inline easybuild/easyconfigs/p/poetry/poetry-1.8.3-GCCcore-13.3.0.eb:168]: checksum

Comment (boegel) [reply easybuild/easyconfigs/p/poetry/poetry-1.8.3-GCCcore-13.3.0.eb]: fixed in 3688e75

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
bear-pg0202u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8480CL (sapphirerapids), Python 3.6.8
See https://gist.github.com/branfosj/5d67e46903548a906fc4b9664283bc39 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
bear-pg0207u28a.bear.cluster - Linux RHEL 8.6, x86_64, AMD EPYC 9554 64-Core Processor (zen4), Python 3.6.8
See https://gist.github.com/branfosj/7ce78064b9324f18ceb85377d995be1a for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/e4b2c5cd4917b188e64c48f1c2bbb943 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/68c3736d36c939f6c6c782d38c96e8df for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
starfive - Linux Debian GNU/Linux n/a, RISC-V-64, UNKNOWN, Python 3.10.9
See https://gist.github.com/bedroge/970102ccba10a385594f38c503613e95 for a full test report.

**Pull request merged**